assets/src/edit-story/components/richText/formatters/italic.js

```diff
@@ -43,7 +43,7 @@ function stylesToCSS(styles) {
 
 function isItalic(editorState) {
   const styles = getPrefixStylesInSelection(editorState, ITALIC);
-  return !styles.includes(NONE);
+  return Boolean(styles.length) && !styles.includes(NONE);
```

Comment:
Wouldn't we expect `styles` here to always contain `NONE` for an empty case?

/cc @barklund

Reply:
I'm pretty sure that should happen. But there might be weird edge cases.